packages/plugins/i18n/admin/src/translations/en.json

```
@@ -6,7 +6,12 @@
   "CMEditViewCopyLocale.copy-failure": "Failed to copy locale",
   "CMEditViewCopyLocale.copy-success": "Locale copied",
   "CMEditViewCopyLocale.copy-text": "Fill in from another locale",
+  "CMEditViewCopyLocale.cancel-text": "No, cancel",
   "CMEditViewCopyLocale.submit-text": "Yes, fill in",
+  "CMEditViewCopyLocale.dialog.title": "Confirmation",
+  "CMEditViewCopyLocale.dialog.body": "Your current content will be erased and filled by the content of the selected locale:",
+  "CMEditViewCopyLocale.dialog.field.label": "Locale",
```

```suggestion
  "CMEditViewCopyLocale.dialog.body": "Your current content will be erased and filled by the content of the selected locale",
```

Is the colon intended it reads a bit weirdly to me